functions/Remove-DbaFirewallRule.ps1

```diff
@@ -0,0 +1,184 @@
+function Remove-DbaFirewallRule {
+    <#
+    .SYNOPSIS
+        Removes firewall rules for SQL Server instances from the target computer.
+
+    .DESCRIPTION
+        Removes firewall rules for SQL Server instances from the target computer.
+        As the group and the names of the firewall rules are fixed, this command
+        only works for rules created with New-DbaFirewallRule.
+
+        This is basically a wrapper around Remove-NetFirewallRule executed at the target computer.
+        So this only works if Remove-NetFirewallRule works on the target computer.
+
+        The functionality is currently limited. Help to extend the functionality is welcome.
+
+        As long as you can read this note here, there may be breaking changes in future versions.
+        So please review your scripts using this command after updating dbatools.
+
+    .PARAMETER SqlInstance
+        The target SQL Server instance or instances.
+
+    .PARAMETER Credential
+        Credential object used to connect to the Computer as a different user.
+
+    .PARAMETER Type
+        Removes firewall rules for the given type(s).
+        Valid values are:
+            Engine - for the SQL Server instance
+            Browser - for the SQL Server Browser
+            AllOnComputer - for all firewall rules on the target computer related to SQL Server
```

Comment:
Just realized that AllOnComputer is too scary, sounds like every firewall rule. Can you say AllSQL or AllDba or SqlRelated or AllInstance or something. Yeah I like AllInstance